src/server/plugins/engine/components/DatePartsField.test.ts

```diff
@@ -1,8 +1,25 @@
 import { ComponentType, type ComponentDef } from '@defra/forms-model'
 
 import { DatePartsField } from '~/src/server/plugins/engine/components/DatePartsField.js'
+import { FormModel } from '~/src/server/plugins/engine/models/FormModel.js'
+import { type FormDefinition } from '~/src/server/plugins/engine/services/formsService.js'
+
+
 
 describe('Date parts field', () => {
+  let formModel: FormModel
+
+  beforeEach(() => {
+    const definition: FormDefinition = {
+      pages: [],
+      lists: [],
+      sections: [],
+      conditions: []
+    }
+
+    formModel = new FormModel(definition, { basePath: '' })
```

Comment:
Mind if keep `basePath: 'test'` I've been using in all the new tests?

Think of it as our test slug

```suggestion
    formModel = new FormModel(definition, {
      basePath: 'test'
    })
```